docs/reference/env-variables.rst

```diff
@@ -0,0 +1,678 @@
+.. meta::
+    :description: Environment variables reference
+    :keywords: AMD, ROCm, environment variables, environment, reference
+
+.. role:: cpp(code)
+   :language: cpp
+
+.. _env-variables-reference:
+
+*************************************************************
+Environment variables reference
+*************************************************************
+
+General overview
```

Comment:
```suggestion
General environment variables
```

Reply:
Already changed locally. I will finish this PR and https://github.com/ROCm/HIP/pull/3515 today.